tests/examples/example_1.py

```diff
@@ -0,0 +1,20 @@
+"""Example: Reference files anywhere """
```

Comment:
I think all the examples should be in a top-level directory `examples/` rather than `tests/examples/` so they are easier to find. But I can also do that once this PR is merged, it's easy enough. As you prefer.

Reply:
I would also put the explanation of the example that is in the README as a docstring inside each example.py file. This way all the relevant info is in one place. Again, I can do this after merging the PR as you prefer.

Reply:
moved, added text to the docstring. Also included just short fragments in the README itself to focus on the highlights.